### PR TITLE
Feat/add playbook reject old alert

### PR DESCRIPTION
**Pull request by Thibault-MAL**
Reject old alerts not updated for 1 year.

### CODE REVIEW

File: playbooks/templates/Reject_old_alerts.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        "1"
+      ],
+      "name": "Foreach",
+      "type": "operator",
+      "items": "[{%for i in range(0, 100)%}{{i}},{%endfor%}]",
+      "outputs": {
+        "default": []
+      },
+      "subtype": "foreach"
+    },
```
Comment (gaelmuller) [inline playbooks/templates/Reject_old_alerts.json:64]: should probably be removed

File: playbooks/templates/Reject_old_alerts.json [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      ],
+      "outputs": {
+        "else": [
+          "10"
+        ],
+        "has_more": []
+      },
+      "subtype": "condition"
+    }
+  },
```
Comment (gaelmuller) [inline playbooks/templates/Reject_old_alerts.json:113]: Not sure I understand the point here. What is the "End of the playbook" action ?

**Review by gaelmuller — CHANGES_REQUESTED** [review]
Ideally for template readibility you chose reorder nodes

**Comment by Thibault-MAL** [comment]
Close in favor of https://github.com/SEKOIA-IO/Community/pull/50

**Pull request closed without merging**